src/configure.ac

```diff
@@ -602,7 +602,7 @@ else
       [AC_MSG_CHECKING([supplied paths for OpenSSL])]
       [JTR_LIST_ADD(CFLAGS, [$OPENSSL_CFLAGS])]
       [if test "x$cross_compiling" = "xyes" -a "x$OPENSSL_LIBS" = "x"; then
-        OPENSSL_LIBS="-lssl -lcrypto"
+        OPENSSL_LIBS="${OPENSSL_LIBS} -lssl -lcrypto"
```

Comment:
This will only happen when `$OPENSSL_LIBS` is empty (per line 604) so it's a no-op change